### PR TITLE
Feat: added storybook

### PR DESCRIPTION
**Pull request by kelvin3127**
-Added Storybook to the application.
-Added Storybook Readme.

How to get storybook running
    Install the required dependencies
        -This can be done using the command ( npm install )
    Run storybook to view the story on local host
        -This can be done using the command ( npm run storybook )

Writing a story
    Each component should have a story and a story should end in component name followed by *.stories.js* 
        -Ie. Button.vue should have Button.stores.js.

### CODE REVIEW

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Make sure to assign yourself to the PR,
Add Reviewers (namely Myself, Mike and the rest of the group)
Do not make PRs against main, PRs are to be merging into dev not main.
Make sure that only the files that need to be added into the repo are added.

File: frontend/src/stories/Button.stories.ts
Comment (YoussefLaunchUp) [inline frontend/src/stories/Button.stories.ts:1]: This entire stories folder needs to be removed

File: frontend/src/stories/assets/code-brackets.svg
Comment (YoussefLaunchUp) [inline frontend/src/stories/assets/code-brackets.svg:1]: This entire assets folder needs to be removed

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Documentation for How to use Storybook (both how to get it up and running as well as how to add new stories is needed)

File: frontend/src/stories/Button.stories.ts
Comment (kelvin3127) [reply frontend/src/stories/Button.stories.ts:1]: > Documentation for How to use Storybook (both how to get it up and running as well as how to add new stories is needed)

Removed Stories Folder and added a Storybook Readme MD File

**Comment by roxanehe** [comment]
Thanks Kelvin!!

File: frontend/src/Stores/counter.ts
Comment (YoussefLaunchUp) [inline frontend/src/Stores/counter.ts:1]: Stores folder should be lowercase.

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
PR looks good, small nit added.

Also the pr title is wrong, it needs to follow conventional commits semantics.

**Review by YoussefLaunchUp — APPROVED** [review]

**Review by neli-rashkova — COMMENTED** [review]
Looks great, thanks Kelvin!

**Pull request merged**